ci-operator/config/openshift/origin/openshift-origin-release-3.11.yaml

```diff
@@ -148,6 +148,11 @@ tests:
   commands: JUNIT_REPORT=1 KUBERNETES_SERVICE_HOST= make verify -k
   container:
     from: bin
+- as: e2e-aws
+  commands: TEST_FOCUS=Suite:openshift/conformance/parallel/minimal TEST_SKIP='(\[local\]|\[templates\]|\[Feature:Router\]|Kubectl
+    logs)' run-tests
+  openshift_ansible_custom:
```

Comment:
Since we are heading toward deprecating the templates, you should create a multi-stage workflow to achieve what you need to test here.

Reply:
The purpose of getting the e2e-aws test in is to supplement the tests that will not be available when we shutdown Jenkins on Aug 3, 2022 (which is coming very soon).  This test is for Openshift 3.11 which is not something we want to spend too much time on given how old it is.  My purpose of making this test using the old templates is to get something running quickly with minimal investment by replicating an already existing and working/trusted e2e-aws test in the openshift/openshift-ansible repo.